### DISCUSSION TITLE
Stats API filter by hostname

### DISCUSSION THREAD

**Discussion by clementmas** [plausible/analytics#2994] [Anything Else]
Is it possible to filter the Stats API results by hostname/domain?

I use wildcard subdomains _{username}.example.com_ and I need to display the number of visits for each user.

I was previously able to do this with GA and it looks like it's supported by the [Fathom API](https://usefathom.com/api#aggregation) (filters > hostname).
You have a great product and I'd like to use it but the lack of support for multiple domains might prevent me from doing so.

Thanks!

**Comment by metmarkosaric** [comment]
this is now possible with `event:hostname`. see https://plausible.io/docs/stats-api#properties
